l10n_it_fatturapa_in/models/account.py

```diff
@@ -215,6 +218,16 @@ def set_einvoice_data(self, fattura):
             'e_invoice_date_invoice': date_invoice,
         })
 
+    def process_negative_lines(self):
```

Comment:
non si può avere una situazione mista?

Reply:
@tafaRU con @stevech091 @marco-marchiori e Corrado in chiamata abbiamo detto che se ricevo una nota di credito con totale negativo e una situazione righe mista, è meglio che il sistema blocchi la validazione perchè così contatto il fornitore e gli chiedo cosa gli passa per la testa